src/components/Figure/utils.js

```diff
@@ -6,7 +6,7 @@ import {
   MAX_WIDTH_MOBILE
 } from '../Center'
 
-export const getResizedSrcs = (src, displayWidth) => {
+export const getResizedSrcs = (src, displayWidth, fillMaxWidth = true) => {
```

Comment:
Nit: I'd call it `enforce` or `set` `MaxWidth`

Reply:
My good ol' Google slang, fill objects all day long :)